webconsole/src/main/java/org/apache/felix/webconsole/internal/misc/FrameworkPropertiesPrinter.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.felix.webconsole.internal.misc;
+
+
+import java.io.PrintWriter;
+import java.util.Map;
+import java.util.TreeMap;
+
+import org.apache.felix.webconsole.internal.AbstractConfigurationPrinter;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.Constants;
+import org.osgi.framework.namespace.NativeNamespace;
+import org.osgi.framework.wiring.BundleCapability;
+import org.osgi.framework.wiring.BundleWiring;
+
+
+public class FrameworkPropertiesPrinter extends AbstractConfigurationPrinter
+{
+
+    private static final String TITLE = "Framework Properties";
+
+    public String getTitle()
+    {
+        return TITLE;
+    }
+
+    public void printConfiguration( PrintWriter printWriter )
+    {
+        getFrameworkProperties().entrySet().stream().forEach( e -> ConfigurationRender.infoLine( printWriter, null, e.getKey(), e.getValue() ));
+    }
+
+    private Map<String,String> getFrameworkProperties() 
+    {
+        Bundle systemBundle = getBundleContext().getBundle( Constants.SYSTEM_BUNDLE_LOCATION );
```

Comment:
You can also use `getBundle(Constants.SYSTEM_BUNDLE_ID)`

Reply:
Done in https://github.com/apache/felix-dev/pull/104/commits/f7535ef846f5cb4e6ca9672116af7e2a1b0354ba